crates/sputnik/src/session.rs

```diff
@@ -11,12 +11,15 @@ use wsl::is_wsl;
 
 use std::collections::HashMap;
 use std::convert::TryFrom;
-use std::env::{self, consts::OS};
+use std::env;
 use std::fmt::Debug;
 use std::time::Duration;
 
 use crate::{Report, SputnikError};
 
+// set timeout to 400 ms to prevent blocking for too long on reporting
```

Comment:
Should this be 4000 ms, or should the following line be 400?
Is there a reason that this number was chosen?

Reply:
that's a good question; I'm not actually sure. It's a copy/paste, but the comment makes it seem like the original intent was 400ms--that tracks, too, given that this is a blocking client

<img width="665" alt="Screenshot 2024-07-02 at 2 53 50 PM" src="https://github.com/apollographql/rover/assets/26738844/4cb5fafa-9651-4092-b7a7-3dc98ad00b15">

I guess I'm in favor of 400ms? what do you think?

Reply:
400ms seems reasonable, but we might even be able to lower it. The p99 for the telemetry endpoint is 30ms.

Reply:
100ms?